ui_framework/src/components/button/_button.scss

```diff
@@ -0,0 +1,123 @@
+// Our base button
+.kuiButton {
+  display: inline-block;
+  appearance: none;
+  cursor: pointer;
+  height: $kuiSizeXXL;
+  text-decoration: none;
+  border: solid 1px $kuiColorPrimary;
+  border-radius: $kuiBorderRadius;
+  color: $kuiColorPrimary;
+  padding: 0 $kuiSizeS;
+  min-width: $kuiSize * 7;
+  text-align: center;
+  font-family: $kuiFontFamily;
+  transition: all $kuiAnimSpeedNormal $kuiAnimSlightBounce;
+  whitespace: nowrap;
+
+  &.kuiButton--small {
+    height: $kuiSizeXL;
+  }
+
+  .kuiButton__icon {
+    vertical-align: -2px;
+    fill: $kuiColorPrimary;
+  }
+
+  &:hover, &:focus {
+    text-decoration: underline;
+    background-color: transparentize($kuiColorPrimary, .9);
+  }
+
+  &:focus {
```

Comment:
This is better, but now when you click the button, the "reaction" is delayed until the mouseup event. This means if your click is a little bit slower, the reaction feels out of sync with your action.

I tried playing with different pseudoselectors (`:active`, `:focus:not(:active)`), but couldn't find a good solution. Personally, I think maybe we could do something different to draw the eye to a focused button, e.g. a [pulse effect](https://codepen.io/olam/pen/zcqea) (not looping, just a single pulse).